code/frameworks/nextjs/src/routing/app-router-provider.tsx

```diff
@@ -101,6 +101,7 @@ export const AppRouterProvider: React.FC<React.PropsWithChildren<AppRouterProvid
                   childNodes: new Map(),
                   tree,
                   url: pathname,
+                  loading: null,
```

Comment:
@valentinpalkovic I had to add this to get TS check to pass.

I think it's due to some dep getting deduped, I wanted to loop you in to have your eyes on it.
Can you check if this is an OK change?